source/jormungandr/jormungandr/interfaces/v1/Places.py

```diff
@@ -351,6 +351,10 @@ def get(self, region=None, lon=None, lat=None):
         if args['disable_geojson']:
             g.disable_geojson = True
 
+        user = authentication.get_user(token=authentication.get_token(), abort_if_no_token=False)
+
+        args['shape'] = json.loads(user.shape) if user and user.shape else None
```

Comment:
why do you move this here?
it means kraken autocomplete will also receive shape in the request?